docs/operators/azure_eventhub_input.md

```diff
@@ -0,0 +1,53 @@
+## `azure_eventhub_input` operator
+
+The `azure_eventhub_input` operator reads logs from Azure Event Hub using [Azure's SDK](https://github.com/Azure/azure-event-hubs-go)
+
+The `azure_eventhub_input` operator will use the `EnqueuedTime` field of the event as the parsed entry's timestamp. If `EnqueuedTime` is not set, `azure_eventhub_input` will use `IoTHubEnqueuedTime` if it is set. All other fields are added to the entry's record.
+
+### Configuration Fields
+
+| Field               | Default                | Description                                                                                   |
+| ---                 | ---                    | ---                                                                                           |
+| `id`                | `azure_eventhub_input` | A unique identifier for the operator                                                          |
+| `output`            | Next in pipeline       | The connected operator(s) that will receive all outbound entries                              |
+| `namespace`         |                        | The Event Hub Namespace                                                                       |
+| `name`              |                        | The Event Hub Name                                                                            |
+| `group`             |                        | The Event Hub Consumer Group                                                                  |
+| `connection_string` |                        | The Event Hub [connection string](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-get-connection-string) |
```

Comment:
Typically put the work `required` in the default column, unless it is optional and there is no default.